recipe/meta.yaml

```diff
@@ -12,7 +12,7 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 0
+  number: 1
```

Comment:
I believe the way forward here is to have noarch in the build section: https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python

Reply:
Ok, I went with what we have for griddataformats https://github.com/conda-forge/griddataformats-feedstock/blob/main/recipe/meta.yaml